src/network/bridge.rs

```diff
@@ -699,7 +716,7 @@ fn create_interfaces(
                 host.set_up(netlink::LinkID::ID(link.header.index))
                     .wrap("set bridge up")?;
 
-                (link.header.index, mac)
+                ((link.header.index, mtu), mac)
```

Comment:
there is no need to use a tuple inside a tuple, three values in a tuple should be valid